README.md

```diff
@@ -1,133 +1,138 @@
 # NUM-CON-MON
 
-CQL Measure for the NUM-CON-MON project, including test data.
+CQL Measure for the NUM-CON-MON project.
 
-## Inhaltsverzeichnis
+---
+
+## **Content**
 
-1. [Beschreibung](#beschreibung)
-1. [Dateien](#dateien)
-1. [Deployment](#deployment-)
+1. [Description](#description)
+2. [Files](#files)
+3. [Deployment](#deployment)
     - [1. Evaluate Measure](#1-evaluate-measure)
     - [2. Upload MeasureReport](#2-upload-measurereport)
-    - [3. Transfer MeasureReport vom DIZ zur HRP](#3-transfer-measurereport-vom-diz-zur-hrp)
-1. [Voraussetzungen](#voraussetzungen)
+    - [3. DSF DataTransfer](#3-dsf-datatransfer)
+4. [Prerequisites](#prerequisites)
+6. [Setting up the Test Environment & Test Data Transfer](#setting-up-the-test-environment--test-data-transfer)
 
 ---
 
-## Beschreibung
+## **Description**
```

Comment:
that's what I did after your last PR, headlines are already bold. no need to mark them as bold here